Cargo.toml

```
@@ -1,17 +1,47 @@
 [package]
 authors = [
+  "Jonas Zaddach <jonas.zaddach@gmail.com",
+  "Zachary Sims <zac@icebergdefender.com>",
   "Scott Hutton <schutton@cisco.com>",
-  "Micah Snyder <micasnyd@cisco.com>",
-  "Jonas Zaddach <jzaddach@cisco.com>",
 ]
 categories = ["api-bindings"]
-description = "Asynchronous API for clamav-sys"
-documentation = "https://docs.rs/clamav-async-rs"
+description = "Async ClamAV bindings for Rust"
 edition = "2021"
-keywords = ["clamav", "libclamav", "antivirus"]
+exclude = ["test_data/*"]
+homepage = "https://github.com/zaddach/clamav-rs"
 license = "GPL-2.0"
 name = "clamav-async"
-repository = "https://github.com/Cisco-Talos/clamav-async-rs"
-version = "0.1.0"
+repository = "https://github.com/zaddach/clamav-rs"
+version = "0.5.5"
```

I see the old `clamav-rs` project is 0.5.5 (https://crates.io/crates/clamav-rs). 
But since this is a new project, can probably start it at `0.1.0`. 

```suggestion
version = "0.1.0"
```

---

Already fixed the old `clamav-rs` references.

I'm somewhat-inclined to import this as version `1.0.0` rather than leaving it in perpetual `0.` limbo.  But `0.5.5` is definitely not correct.

---

That's a fair point.  I too prefer projects to stay 0.1 for pre-release and 1.0 for first release to be used in production. We can bump to 1.0 soon as you're ready.